pkg/aws/commandbuilder/commandbuilder.go

```diff
@@ -0,0 +1,126 @@
+package commandbuilder
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+const PARAM_LINE_SEPARATOR = " \\\n"
```

Comment:
NP: sorry for not noticing this earlier, but the convention we have in this project is not using `ALL_CAPS` for constant names. (also for the other consts below)